app/services/model_controller/notifier.rb

```diff
@@ -11,43 +13,28 @@ def success_action
     def success_insert(item)
       respond_to do |format|
         format.html { redirect_to admin_edit_uri(item) }
-        format.json {
-          if params.has_key?(:create_and_return)
-            data = { :redirect_to => admin_list_uri(item.class) }
-          else
-            data = { :form_action => admin_update_uri(item) }
-          end
-          success_message(item, I18n.t('rademade_admin.success_insert_message'), data)
-        }
+        format.json { success_message(item, I18n.t('rademade_admin.success_insert_message'), success_data) }
```

Comment:
Line is too long. [108/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.